apps/pollbook/backend/src/peer_store.ts

```diff
@@ -271,6 +228,31 @@ export class PeerStore extends Store {
         machineId: pollbookService.machineId,
       })
     );
+
+    if (
+      pollbookService.machineId === this.machineId &&
+      pollbookService.status === PollbookConnectionStatus.LostConnection
+    ) {
+      // We are now offline, we should mark all other services as offline.
+      for (const [otherPollbookName, otherPollbookService] of Object.entries(
+        this.connectedPollbooks
+      )) {
+        if (otherPollbookName === avahiServiceName) {
+          continue;
+        }
+        if (
+          CommunicatingPollbookConnectionStatuses.includes(
+            otherPollbookService.status
```

Comment:
do we want to leave peers in `ShutDown` status? is it more accurate to say they're `LostConnection` (and we don't know if they are still powered down or have been powered up)? or does it not really make a difference?